app/src/navigation/screens/ClaimScreen.tsx

```diff
@@ -106,7 +106,15 @@ const ClaimScreen: React.FC = () => {
 
   const onSave = async () => {
     setLoading(true);
-    await addClaim(claim.type, formState, selectedFileIds);
+    let newFormState = formState;
+    if (claim.type === "EmailCredential") {
+      newFormState = {
```

Comment:
This can probably be reformatted to a `map` function if `formState` is an array. If it's an object you can use `Object.entries(formState)` and map that.